openy.install

```diff
@@ -40,3 +40,15 @@ function openy_update_8003() {
     $importer->import('openy_demo_node_landing');
   }
 }
+
+/**
+ * Enable Alerts.
+ */
+function openy_update_8004() {
```

Comment:
Please merge with master and set 8002. @hamrant has been fixed number for his hook_update_N https://github.com/ymcatwincities/openy/pull/181#pullrequestreview-23729058.